pallets/subtensor/tests/networks.rs

```diff
@@ -66,19 +67,19 @@ fn test_remove_uid_for_network() {
         let netuid: u16 = 1;
         let tempo: u16 = 13;
         add_network(netuid, tempo, 0);
-	register_ok_neuron( 1, 55, 66, 0);
+	register_ok_neuron( 1, U256::from(55), U256::from(66), 0);
         let neuron_id ;
-        match SubtensorModule::get_uid_for_net_and_hotkey(netuid, &55) {
+        match SubtensorModule::get_uid_for_net_and_hotkey(netuid, &U256::from(55)) {
             Ok(k) => neuron_id = k,
             Err(e) => panic!("Error: {:?}", e),
         } 
-        assert!(SubtensorModule::get_uid_for_net_and_hotkey(netuid, &55).is_ok());
+        assert!(SubtensorModule::get_uid_for_net_and_hotkey(netuid, &U256::from(55)).is_ok());
         assert_eq!(neuron_id, 0);
-        register_ok_neuron( 1, 56, 67, 300000);
-        let neuron_uid = SubtensorModule::get_uid_for_net_and_hotkey(netuid, &56).unwrap();
+        register_ok_neuron( 1, U256::from(56), U256::from(67), 300000);
+        let neuron_uid = SubtensorModule::get_uid_for_net_and_hotkey(netuid, &U256::from(56)).unwrap();
         assert_eq!(neuron_uid, 1);
         assert_ok!(SubtensorModule::do_remove_network(<<Test as Config>::RuntimeOrigin>::root(), netuid));
-        assert!(SubtensorModule::get_uid_for_net_and_hotkey(netuid, &55).is_err());
+        assert!(SubtensorModule::get_uid_for_net_and_hotkey(netuid, &U256::from(55)).is_err());
```

Comment:
just a general idea for _potential_ modifications of the same value.

If we encapsule this in a function the call this function `U256::from`, next changes will impact few lines and this behaviour is more localized in a single function instead of being all over the place.

As I said, i do not know if this makes sense in the context of the project or Rust, but I share with you this idea in case you think we can have another change like this eventually @SaMotlagh

Reply:
Maybe something like a `AccountId::from_num()`

Reply:
thanks, that's right! we could have sth like from_num(). This way is probably simpler to read as conversion is explicit though. If we need to change type in future again, I will wrap it inside a function.